velox/exec/WindowFunction.h

```diff
@@ -31,6 +31,23 @@ struct WindowFunctionArg {
   std::optional<const column_index_t> index;
 };
 
+/// The scope for calculating the window function. kRows indicates that the
```

Comment:
It would be helpful to move documentation of the individual values next to the values themselves.

```
   /// Calculation may start only after all rows within a partitions are available.
   kPartition,

   /// Calculation may being as soon as ...
   kRows,
````

The name 'Scope' is a bit too generic. Consider moving this enum inside WindowFunctionMetadata or renaming.

Reply:
@mbasmanova Change Scope to ProcessedUnit.